assets/config.yml

```diff
@@ -1312,6 +1312,18 @@ services:
         url: "https://apps.microsoft.com/detail/9NBLGGH2JHXJ"
         target: "_blank" # optional html a tag target attribute
 
+      - name: "Prism Launcher"
+        logo: "assets/tools/Prism.png"
+        subtitle: "a custom launcher for Minecraft"
+        url: "https://prismlauncher.org/download/"
+        target: "_blank" # optional html a tag target attribute
+
+      - name: "Minecraft Java Edition (with ARM Support)"
+        logo: "assets/tools/MinecraftJave.png"
```

Comment:
The image file name is spelled wrong, also thanks for shoutout. :P

Reply:
Thank you for pointing out the issue. I will change the image name for the next submission